### PR TITLE
4.3: VM deployment adjustments

### PR DESCRIPTION
**Pull request by keichwa**
**For rendered output (HTML),** see https://github.com/SUSE/spacewalk/issues/25125

* Update info on KVM and VMware
* Move boot image configuration such as ignition to new file install-ignition.adoc
* Create new file install-ignition.adoc and add to nav

### CODE REVIEW

File: modules/installation-and-upgrade/pages/install-ignition.adoc [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+[IMPORTANT]
+====
+{productname} VM images are prepared with JeOS firstboot configuration dialogs.
+Using a boot image configuration tool such as [systemitem]``Ignition`` is optinal now.
+====
+
+
+
+[[sumavm.ignition]]
+== {productname} basic configuration using [systemitem]``Ignition``
```
Comment (0rnela) [inline modules/installation-and-upgrade/pages/install-ignition.adoc:18]:
```suggestion
== {productname} Basic Configuration Using [systemitem]``Ignition``
```

File: modules/installation-and-upgrade/pages/install-vm.adoc [hunk 2]
```diff
@@ -3,18 +3,28 @@
 
 
 
+This chapter provides procedures how you can deploy a Kernel Virtual Machine (KVM) or VMware image.
```
Comment (0rnela) [inline modules/installation-and-upgrade/pages/install-vm.adoc:6]:
```suggestion
This chapter provides procedures for deploying a Kernel Virtual Machine (KVM) or VMware image.
```

File: modules/installation-and-upgrade/pages/install-vm.adoc [hunk 3]
```diff
@@ -242,25 +90,34 @@ Also consider to create a separate partition for [path]``/srv`` where Kiwi image
 | VirtIO Disk 3        | pgsql     | 100{nbsp}GB
 | VirtIO Disk 4        | swap      | 4{nbsp}GB
 |===
++
 
-. Click btn:[Add Hardware] to attach a virtual CDROM device with the prepared [systemitem]``Ignition`` or [systemitem]``Cloud Init`` disk.
 . Click btn:[Begin Installation] to boot the new VM from the {productname} image.
-  Wait until the login prompt is presented.
-  Log in using credentials set by configuration disk.
+. At the JeOS Firstboot screen select start to continue and check the following configuration dialogs (keyboard layout, license agreement, time zone, password for root).
```
Comment (0rnela) [inline modules/installation-and-upgrade/pages/install-vm.adoc:96]:
```suggestion
. At the JeOS Firstboot screen select [literal]``start`` to continue and check the following configuration options (keyboard layout, license agreement, time zone, password for root).
```

File: modules/installation-and-upgrade/pages/install-vm.adoc [hunk 4]
```diff
@@ -242,25 +90,34 @@ Also consider to create a separate partition for [path]``/srv`` where Kiwi image
...
+
+. When installation completes log in as root.
```
Comment (0rnela) [inline modules/installation-and-upgrade/pages/install-vm.adoc:98]:
```suggestion
. When installation completes, log in as root.
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

**Review by jcayouette — APPROVED** [review]
I have not tested this, but overall looks good.

File: modules/installation-and-upgrade/pages/install-vm.adoc [hunk 5]
```diff
@@ -242,25 +90,34 @@ Also consider to create a separate partition for [path]``/srv`` where Kiwi image
...
 
-. Upload the prepared Ignition or Cloud Init disk file you created using the instructions above.
+. Make a copy of uploaded [filename]``.vmdk`` file using VMware web interface.
+  This will convert provided [filename]``.vmdk`` file to the format suitable for vSpere hypervisor.
 
 . Create and name a new virtual machine based on the Guest OS Family [systemitem]``Linux`` and Guest OS Version {sle} 15 (64-bit).
++
 
+// FIXME: Do we need the following step?  I think, in 5.0 we removed it
 . In [guimenu]``Customize settings`` browse to the uploaded [filename]``.vmdk`` using [systemitem]``IDE controller 0`` storage device in the [systemitem]``Controller Location``.
```
Comment (aaannz) [inline modules/installation-and-upgrade/pages/install-vm.adoc:121]: Not needed, please remove.

File: modules/installation-and-upgrade/pages/install-vm.adoc [hunk 6]
```diff
@@ -242,25 +90,34 @@ Also consider to create a separate partition for [path]``/srv`` where Kiwi image
...
 
 .Procedure: Creating the VMware Virtual Machine
 
-. Download {productname} Server [filename]``.vmdk`` file then transfer a copy to your VMware storage.
+. Download {productname} Server [filename]``.vmdk`` file.
+  Then transfer a copy to your VMware storage.
 
-. Upload the prepared Ignition or Cloud Init disk file you created using the instructions above.
+. Make a copy of uploaded [filename]``.vmdk`` file using VMware web interface.
+  This will convert provided [filename]``.vmdk`` file to the format suitable for vSpere hypervisor.
```
Comment (aaannz) [inline modules/installation-and-upgrade/pages/install-vm.adoc:115]: Maybe add clarification `Use this new copy as a base image for the virtual machine`

**Pull request merged**